### DISCUSSION TITLE
Apostrophe 2.X links?

### DISCUSSION THREAD

**Discussion by hexitex** [apostrophecms/apostrophe#3216] [General]
I understand the need to get new people onto version 3, but could we still have clear links to version 2 please. I can't seem to find them?

**Comment by abea** [comment]
Where are you looking? The default README here is in the version 3 main branch, so it is focused on A3. A2 is still maintained in the `2.0` branch: https://github.com/apostrophecms/apostrophe/tree/2.0, including its README with links. The A3 docs also have a "Versions" switcher in the top navigation to change to A2 docs.

**Reply by abea** [reply]
I suggest you generally use https://v2.docs.apostrophecms.org/ as your source for A2 information and links. If there's something missing there that you use often let us know.

**Reply by hexitex** [reply]
Many thanks,

I was thinking a link on the main apos repo page to version 2. I'm just about to release a nifty boilerplate based on 2 that has a few nice widgets: 

> Diagrams.net plugin - add and edit industry standard diagrams in a click. You can even make collages from images. Full SVG support so nice and small diagrams.

> Smart div widget with:
transitions, parallax, image filters, borders, shadows and video backgrounds and lots more all from a widget interface.

The real killer is templates from pieces - build templates from other templates and widgets without destroying the original piece. The markup is recreated in a standard widget and coupled with smart div's is making building sites a pleasure for non coders.

**Reply by abea** [reply]
That sounds very cool. Let us know if you decide to open source them. We're looking to start getting more community modules on the [extensions page](https://apostrophecms.com/extensions).

Regarding the version thing, if you haven't already you might consider adding a `peerDependencies` section to the package.json files to specify A2. I guess your boilerplate will use that already, so it's only an issue if they are published separately.
